app/Filament/Resources/EmailTemplateResource/Pages/EditEmailTemplate.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Filament\Resources\EmailTemplateResource\Pages;
+
+use Filament\Forms\Form;
+use Filament\Actions\DeleteAction;
+use Filament\Support\Colors\Color;
+use Filament\Forms\Components\Select;
+use Filament\Forms\Components\TextInput;
+use Filament\Resources\Pages\EditRecord;
+use App\Filament\Resources\EmailTemplateResource;
+use Filament\Forms\Components\SpatieMediaLibraryFileUpload;
+
+class EditEmailTemplate extends EditRecord
+{
+    protected static string $resource = EmailTemplateResource::class;
+
+    public function form(Form $form): Form
+    {
+        return $form
+            ->schema([
+                TextInput::make('name')
+                    ->string()
+                    ->required()
+                    ->autocomplete(false),
+                Select::make('primary_color')
+                    ->native(false)
+                    ->allowHtml()
```

Comment:
Why allowHtml?

Reply:
I was playing with a color indicator and I forgot to remove it after I scrapped the idea.